pkg/api/handler.go

```diff
@@ -5,25 +5,23 @@ import (
 
 	mgo "gopkg.in/mgo.v2"
 
-	"encoding/json"
-
 	"github.com/hellofresh/janus/pkg/errors"
+	"github.com/hellofresh/janus/pkg/notifier"
 	"github.com/hellofresh/janus/pkg/opentracing"
 	"github.com/hellofresh/janus/pkg/request"
 	"github.com/hellofresh/janus/pkg/response"
 	"github.com/hellofresh/janus/pkg/router"
-	"github.com/hellofresh/janus/pkg/store"
 )
 
 // Controller is the api rest controller
 type Controller struct {
-	repo      Repository
-	publisher store.Publisher
+	repo     Repository
+	notifier *notifier.Notifier
 }
 
 // NewController creates a new instance of Controller
-func NewController(repo Repository, publisher store.Publisher) *Controller {
-	return &Controller{repo, publisher}
+func NewController(repo Repository, notifier *notifier.Notifier) *Controller {
```

Comment:
`notifier.Notifier` could be a interface where `notifier.New(...)` returns it instead of the struct, just to facilitate tests, but it can be done in other PR if you prefer